config/images.yml

```diff
@@ -36,8 +39,8 @@
     class: Shopsys\FrameworkBundle\Model\Category\Category
     sizes:
         -   name: ~
-            width: 70
-            height: 70
+            width: 30
+            height: 30
```

Comment:
hmm, why did you change this values?

Reply:
Fixed in 268abd1bae86f02635c43016b0523475fff3663c